### DISCUSSION TITLE
mssql-jdbc-14.4.2.jre8 TLSv1.3

### DISCUSSION THREAD

**Discussion by ovidiugugila** [microsoft/mssql-jdbc#2312] [General]
Hi, everyone,

I've the following scenario.

I recently had a scenario in my app where is required to use TLSv1.3 in my client-server communication.
I use SQL Management Studio 2022 in order to use TLSv1.3 and as client I use a java client. My problem is the following:

 - when I build the connection url in order to communicate with database even if I set the protocol to TLSv1.3 in my ClientHello I can see the both version of TLS is supported, TLSv1.2 and also TLSv1.3.

I already set the sslProtocol=TLSv1.3;encrypt=strict;trustServerCertificate=false; in my connection url.

Is any way to force the Client side to use the TLSv1.3 only?

![connUrlFlags](https://github.com/microsoft/mssql-jdbc/assets/22844582/fe6c9d2e-07c8-4084-88c7-44a23f88af15)
![gitSupportedVersion](https://github.com/microsoft/mssql-jdbc/assets/22844582/755b9fe2-1faf-414d-8534-ab5757def14c)

**Comment by Jeffery-Wasty** [comment]
This is a duplicate of https://github.com/microsoft/mssql-jdbc/issues/2313. Any discussion for this issue will take place on that page.
